scripts/bert/finetune_classifier.py

```diff
@@ -183,6 +201,20 @@
     model.classifier.initialize(init=mx.init.Normal(0.02), ctx=ctx)
     loss_function = gluon.loss.SoftmaxCELoss()
 
```

Comment:
Is the option tested? Pretrained bert params include extra layers, model.bert.load_params may give a warning when extra decoder and classifier params are provided. Use `ignore_extra=False` in `model.bert.load_parameters `. 

I would set `pretrained=False` in `get_bert_model` if either `pretrained_bert_parameters` or `model_parameters` is not None. 

I would also skip regression/classifier.intialization if `model_parameters` if provided.

Reply:
Already corrected.